src/Tribe/Repositories/Event.php

```diff
@@ -216,6 +190,30 @@ public function __construct() {
 		$this->add_simple_tax_schema_entry( $tribe_events_category . '_not_in', $tribe_events_category, 'term_not_in' );
 		$this->add_simple_tax_schema_entry( 'tag', 'post_tag' );
 		$this->add_simple_tax_schema_entry( 'tag_not_in', 'post_tag', 'term_not_in' );
+
+		add_action('pre_get_posts', [ $this, 'ensure_event_post_types' ], 50 );
```

Comment:
This should be part of a Hooks FIle, otherwise this will be hooked multiple times. Also spacing.

Reply:
I put it in the ORM service provider?